README.md

```diff
@@ -4,23 +4,23 @@
 
 Utility Functions and Development Tools for the Admiral Package Family
 
-<!-- badges: start -->
+<!-- start badges -->
```

Comment:
@bms63 , this makes the badges appear on both the main and the dev site:

![image](https://github.com/user-attachments/assets/f84dc1e4-4d7b-4b04-9692-d0ac90002087)

What do you think?

Reply:
looks great to me!! I will adjust!